### DISCUSSION TITLE
Annotation can not be SHOW in AR Quicklook in iOS Device

### DISCUSSION THREAD

**Discussion by basiirzulkifle** [google/model-viewer#4462] [Q&A] [answered]
I am using ios 16. It worked on Android. But not in IOS. Any thing that I missed out to setting? 

I am using this code . `<style>
  .hotspot{
    display: block;
    width: 20px;
    height: 20px;
    border-radius: 10px;
    border: none;
    background-color: blue;
    box-sizing: border-box;
    pointer-events: none;
  }

  .hotspot[slot="hotspot-hand"]{
    --min-hotspot-opacity: 0;
    background-color: red;
  }

  .hotspot[slot="hotspot-foot"]:not([data-visible]) {
    background-color: transparent;
    border: 3px solid blue;
  }

  .annotation{
    background-color: #888888;
    position: absolute;
    transform: translate(10px, 10px);
    border-radius: 10px;
    padding: 10px;
  }
  /* This keeps child nodes hidden while the element loads */
  :not(:defined) > * {
    display: none;
  }
</style>
<model-viewer id="hotspot-demo" ar ar-modes="webxr" camera-controls touch-action="pan-y" src="../../shared-assets/models/Astronaut.glb" poster="../../assets/poster-astronaut.webp" shadow-intensity="1" alt="A 3D model of an astronaut.">
  <button class="hotspot" slot="hotspot-visor" data-position="0 1.75 0.35" data-normal="0 0 1"></button>
  <button class="hotspot" slot="hotspot-hand" data-position="-0.54 0.93 0.1" data-normal="-0.73 0.05 0.69">
    <div class="annotation">This hotspot disappears completely</div>
  </button>
  <button class="hotspot" slot="hotspot-foot" data-position="0.16 0.1 0.17" data-normal="-0.07 0.97 0.23" data-visibility-attribute="visible"></button>
  <div class="annotation">This annotation is fixed in screen-space</div>
</model-viewer>`

**Comment by elalish** [comment, marked as answer]
Correct, since iOS does not support WebXR, this is currently impossible. :( FYI @grorg.
